english/gitnew/gitnew.md

```diff
@@ -0,0 +1 @@
+click[here](www.google.com) to open google website
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.